src/cli/ui.py

```diff
@@ -91,14 +91,15 @@ def toggle_data(self):
         title()
 
         data_options = [
-            ("C", "CPU"),
-            ("G", "GPU"),
-            ("B", "Motherboard" if system().lower() != "darwin" else "Vendor"),
-            ("M", "Memory"),
-            ("N", "Network"),
-            ("A", "Audio"),
-            ("I", "Input"),
-            ("S", "Storage"),
+            ("C", self.langparser.parse_message("src-cli-ui-cpu")),
+            ("G", self.langparser.parse_message("src-cli-ui-gpu")),
+            ("B", self.langparser.parse_message("src-cli-ui-motherboard") if system().lower() != "darwin"
```

Comment:
We could alternatively rename this to `baseboard`, as I believe that's more of a unified terminology, so there wouldn't be a need to have this conditional branching here.

Reply:
will keep this in mind.